src/views/pacbio/PacbioRuns.vue

```diff
@@ -1,28 +1,124 @@
+                      type="search"
+                      id="filterInput"
+                      placeholder="Type to Search">
+        </b-form-input>
+        <b-input-group-append>
+          <b-button :disabled="!filter" @click="filter = ''">Clear</b-button>
+        </b-input-group-append>
+      </b-input-group>
+    </b-form-group>
+    <br>
+
+    <b-table id="runs-table"
+          hover
+          show-empty
+          :items="runs"
+          :fields="fields"
+          :filter="filter"
+          :sort-by.sync="sortBy"
+          :sort-desc.sync="sortDesc"
+          :per-page="perPage"
+          :current-page="currentPage"
+          @filtered="onFiltered">
+
+        <template v-slot:cell(actions)="row">
+          <a :id="generateId('generate-sample-sheet', row.item.id)"
+             :href="generateSampleSheetPath(row.item.id)">
+             Generate Sample Sheet
+          </a>
+        </template>
+    </b-table>
+
+    <span class="font-weight-bold">Total records: {{ runs.length }}</span>
+
+    <div>
+      <b-pagination class="float-right"
+              v-model="currentPage"
+              :total-rows="runs.length"
+              :per-page="perPage">
+      </b-pagination>
+    </div>
+
   </div>
 </template>
 
 <script>
+import Alert from '@/components/Alert'
+import Helper from '@/mixins/Helper'
+import TableHelper from '@/mixins/TableHelper'
+
+import { createNamespacedHelpers } from 'vuex'
+const { mapActions, mapGetters } = createNamespacedHelpers('traction/pacbio/runs')
 
 export default {
   name: 'PacbioRuns',
-  mixins: [],
+  mixins: [Helper, TableHelper],
   props: {
   },
   data () {
     return {
+      fields: [
+        { key: 'id', label: 'Run ID', sortable: true },
+        { key: 'name', label: 'Name', sortable: true },
+        { key: 'state', label: 'State', sortable: true },
+        { key: 'template_prep_kit_box_barcode', label: 'Template Prep Kit BB', sortable: true },
+        { key: 'binding_kit_box_barcode', label: 'Binding Kit BB', sortable: true },
+        { key: 'template_prep_kit_box_barcode', label: 'Template Prep Kit BB', sortable: true },
+        { key: 'sequencing_kit_box_barcode', label: 'Sequencing Kit BB', sortable: true },
+        { key: 'dna_control_complex_box_barcode', label: 'DNA Control Complex BB', sortable: true },
+        { key: 'system_name', label: 'System Name', sortable: true },
+        { key: 'created_at', label: 'Created at', sortable: true },
+        { key: 'actions', label: 'Actions' },
+      ],
+      filteredItems: [],
+      filter: null,
+      sortBy: 'created_at',
+      sortDesc: true,
+      perPage: 5,
+      currentPage: 1,
     }
   },
   methods: {
+    async provider() {
+      try {
+        await this.setRuns()
+      } catch (error) {
+        this.showAlert("Failed to get runs: " + error.message, 'danger')
+      }
+    },
+    generateId(text, id) {
+      return `${text}-${id}`
+    },
+    generateSampleSheetPath(id) {
```

Comment:
I noticed that you put this here rather than adding it to the config. I remember we had a brief discussion about this being difficult to add to the request model. Did you have a look at that? Although we can get away it here I am pretty sure that when we come to tracking for PacBio there will be much more of this. Any thoughts on what we can do?